apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -51,17 +51,17 @@ Page {
                 readonly property bool _homePageBlank: homePageConfig.value === "about:blank"
 
                 width: parent.width
-                //: Label for home page text field
-                //% "Home Page"
-                label: qsTrId("settings_browser-la-home_page")
-                //% "Default"
-                value: _homePageBlank ? qsTrId("sailfish_browser-la-home_page_default") : removeProtocolTypeFromUri(homePageConfig.value)
+                //: Label for start view text field
+                //% "Start view"
+                label: qsTrId("settings_browser-la-start_view")
+                //% "Start view"
```

Comment:
Not sure whether "Home page" should be renamed exactly to "Start view" which has 2 possible values: "Start view" and "Web page".

Reply:
Yeap, I think this should stay as "Home page" but I'd rather not use "page" word with "start view" that is the qml thingie. Reason being that it easily confuses people when there are "page" when referring to web pages and same for "views".